src/components/atoms/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,15 @@
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import MuiBreadcrumbs, {
+  BreadcrumbsProps as MuiBreadcrumbsProps,
+} from '@mui/material/Breadcrumbs';
+
+export type BreadcrumbsProps = MuiBreadcrumbsProps;
```

Comment:
I'd remove this and just use `MuiBreadcrumpsProps`, we should also do this for other components